docs/reference/modules/cluster/remote-clusters-security.asciidoc

```diff
@@ -0,0 +1,50 @@
+[[remote-clusters-security]]
+=== Configure remote clusters with security
+To use {ccr} or {ccs} safely with remote clusters, enable security on all 
+connected clusters and configure Transport Layer Security (TLS) on every node.
+Configuring TLS security at the transport level is minimally required for
+remote clusters. For additional security, configure TLS at the
+<<security-basic-setup-https,HTTP level>> as well.
+
+All connected clusters must trust one another and be mutually authenticated at
+the TLS level on at least the transport layer. This means that the local cluster
+trusts the certificate  authority (CA) of the remote cluster, and the remote
+cluster trusts the CA of the local cluster. When establishing a connection, all
+nodes will verify certificates from the nodes they are connecting to. This 
```

Comment:
```suggestion
nodes will verify certificates from the nodes on the other side. This 
```

It is not just "connecting to". It also happens for "connecting from" if that makes sense. Basically the verfication is mutual. regardless which side initiates it.